aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -396,6 +419,51 @@ impl AptosVM {
         }
     }
 
+    fn must_create_account_if_does_not_exist(
+        &self,
+        session: &mut SessionExt,
+        gas_meter: &mut impl AptosGasMeter,
+        txn_data: &TransactionMetadata,
+        resolver: &impl AptosMoveResolver,
+        log_context: &AdapterLogSchema,
+        change_set_configs: &ChangeSetConfigs,
+    ) -> Result<(), VMStatus> {
+        // Use this session to perform the write operations
+        create_account_if_does_not_exist(session, &mut UnmeteredGasMeter, txn_data.sender())
```

Comment:
not sure I understand why we need this part, can't we just spawn the storage_session (I'd call it cleanup_session) and return the respawned session back for epilogue? 

I think we can even unify it with success_transaction_cleanup (and maybe rename them to match try-catch-final)